packages/components/src/range-control/README.md

```diff
@@ -193,6 +229,86 @@ The maximum value accepted. If higher values are inserted onChange will not be c
 - Required: No
 - Platform: Web | Mobile
 
+#### renderTooltipContent
+
+A way to customize the rendered UI of the value. Example:
+
+```jsx
+const customTooltipContent = value => `${value}%`
+
+const MyRangeControl() {
+	return (<RangeControl renderTooltipContent={ customTooltipContent } />)
+}
+```
+
+- Type: `Function`
+- Required: No
+- Platform: Web
+
+#### showTooltip
+
+Forcing the Tooltip UI to show or hide.
+
+- Type: `Boolean`
+- Required: No
+- Platform: Web
+
+#### step
+
+The stepping interval between `min` and `max` values. Step is used both for user interface and validation purposes.
+
+- Type: `Number`
+- Required: No
+- Platform: Web
+
+#### tooltipPosition
+
+Determines the render position of the Tooltip. Either `auto` (default), `top`, or `bottom`.
+
+- Type: `String`
+- Required: No
+- Platform: Web
+
+#### tooltipTimeout
+
+Determines the `timeout` (ms) to hide the Tooltip based on mouse movements.
+
+- Type: `Number`
+- Required: No
+- Platform: Web
+
+#### tooltipZIndex
+
+Determines the CSS `z-index` property of the Tooltip.
+
+- Type: `Number`
+- Required: No
+- Platform: Web
+
+#### value
+
+The current value of the range slider.
+
+- Type: `Number`
+- Required: Yes
+- Platform: Web | Mobile
+
+#### width
```

Comment:
Do we need the width prop or can the component be like 100% by default?

Reply:
Ah! Good catch. This prop isn't required. Will make that change.